### PR TITLE
Setup full configuration, each config could be setup from environment

### PR DESCRIPTION
**Pull request by mrproliu**
Mostly is update the gRPC client configuration

### CODE REVIEW

File: configs/satellite_config.yaml [hunk 1]
```diff
@@ -37,8 +37,55 @@ telemetry:
 sharing:
   clients:
     - plugin_name: "grpc-client"
+      # The gRPC server address finder type
+      finder_type: ${SATELLITE_GRPC_CLIENT_FINDER:static}
```
Comment (wu-sheng) [inline configs/satellite_config.yaml:41]: It is better to list all options here.

File: docs/en/setup/plugins/client_grpc-client.md [hunk 2]
```diff
@@ -30,6 +45,7 @@ check_period: 5
 ## Configuration
 |Name|Type|Description|
 |----|----|-----------|
+| finder_type | string | The gRPC server address finder type |
```
Comment (wu-sheng) [inline docs/en/setup/plugins/client_grpc-client.md:48]: Please describe how to activate different finders.

**Comment by wu-sheng** [comment]
I remember you used to add k8s service discovery, do I remember wrongly or something?

**Comment by wu-sheng** [comment]
OK, then, please make the config more clear about how to activate k8s upstream service discovery.

File: docs/en/setup/plugins/client_grpc-client.md (hunk 2 above)
Comment (mrproliu) [reply docs/en/setup/plugins/client_grpc-client.md:48]: been added to the example file.

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**